pkg/endpoint/kubernetes/cep.go

```diff
@@ -0,0 +1 @@
+package kubernetes
```

Comment:
What's the point of this file?

Reply:
Will remove, it's a leftover. Sorry about that.